brainstormforce/astra
# AST-2813 - fix: Content bg color not applying to posts / pages when Narrow conta…

…iner selected

### Description

- Content background color does not apply in customizer preview when Narrow Width Layout is selected.
- The preview does not work for Blog / Posts & pages when Narrow Width Layout is selected.
- JIRA - https://brainstormforce.atlassian.net/browse/AST-2813
### Screenshots

https://d.pr/v/h0rvst
### Types of changes


bugfix: Narrow width content background preview is not working for page and post



### How has this been tested?

- Select Narrow Width Layout from Single Blog Posts or Pages.
- Apply Content Bg Color from Customizer.
- Check in Customizer Preview.

### Checklist:


[AST-2813]: https://brainstormforce.atlassian.net/browse/AST-2813?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
